proposals/distributed-search.md

```diff
@@ -0,0 +1,62 @@
+# Proposal: Search of Distributed Repositories
+
+Repositories in Helm were designed to be distributed. Instead of a central repository, the concept was for many distributed repositories to exist. This was modeled after [Debian APT](https://en.wikipedia.org/wiki/APT_(Debian)).
+
+To kick start charts, the charts repository was created. Eventually there were the stable and incubator repositories with Helm automatically adding the stable repository. To visually view repositories the monocular project was created which was used to launch what is now hub.kubeapps.com. This site enabled the search of the stable and incubator repositories. With Helm's use of stable repository and the UI search focusing on the community provided charts the community as a whole moved towards a more centralized approach. When companies or others wanted to share their projects they put them into the community charts.
+
+This proposal is about enabling distributed repositories by making them and their packages discoverable.
+
+## A Single Search Location
+
+The goal of this is to have a search site and API that enables the search of many public repositories. Private repositories are a separate scope and can operate with existing tools.
```

Comment:
Might be good to specifically state hosting repositories as a non-goal.

Reply:
Good idea. Updated.